bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -86,6 +87,17 @@ impl fmt::Display for PackedLockTime {
     }
 }
 
+impl FromHexStr for PackedLockTime {
+    type Error = core::num::ParseIntError;
```

Comment:
I'm having trouble implementing this. Can't figure out how to convert from the std/core `ParseIntError` to `parse::ParseIntError`. I've tried implementing From, which became a pain because of the generic, every type of map_err I could think of, and more. I believe I went to implement this at one point but ran into similar issues. I don't know how to resolve this and just opted to use core::num::ParseIntError.

Removed the String from FromHexError::ParseHex

Reply:
Also, crate::parse::ParseIntError holds the core::num::ParseIntError, and from_str_radix returns std::num::ParseIntError

Reply:
I suggest for starters add `pub(crate) fn hex_u32` in `parse` module that is similar to `parse::int` and call `crate::parse::hex_u32` instead of `u32::from_str_radix` in all those places. We can clean it up (make it generic) later.

Reply:
for any type `path::Foo` `core::path::Foo` is always the same type as `std::path::Foo` so don't worry about it, just write `core` instead of `std` everywhere.

Reply:
Ok, that makes sense. Thanks for clarifying

Reply:
Hey Connor, you've pushed a new version which implies request for review but you haven't addressed the comment here by @Kixunil about using `crate::parse`. Please either implement his suggestions or comment on why you think this is not a good idea. Definitely do not blindly follow suggestions, anyone is welcome to argue their opinion but we need either a counter-argument or the suggested change to move forward. This is standard procedure in many open source projects. "I don't understand what you are saying here so I left it as is" is also a valid response :)

Reply:
I'm in the process of implementing the change. Once its squashed, it'll be good to go. Pushing is force of habit so I'll hold back the commits as to not request reviews

Reply:
No sweat, legend.